src/Components/Team/Team.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import logo from "./../../asset/teamlogo.jpg";
+import { teamData } from "./teamData";
+import CardComponent from "./Card";
+import SearchBar from "../SearchBar";
+
+const TeamComponent = () => {
+  return (
+    <>
+      <TeamHeader></TeamHeader>
+      <TeamBody></TeamBody>
+    </>
+  );
+};
+
+const TeamHeader = () => {
+  return (
+    <div className="wrapper_logo">
+      <img className="header_logo" src={logo}></img>
+    </div>
+  );
+};
+
+const CardContainer = ({ filteredResult }) => {
```

Comment:
can make separate file for different containers. It will make Code refactoring easier in future.